src/components/SusPage.tsx

```diff
@@ -86,16 +86,40 @@ export class SusPage extends React.Component<susPageProps, susPageState> {
           <div className="grid grid-rows-[auto,1fr] gap-2 justify-center w-full min-w-[1300px]">
             <div>
               {this.props.type === "Static" ? (
-                <h2 className="text-2xl font-bold mb-8 text-center">
+                <h2 className="text-2xl font-bold mb-8 text-center mt-8">
                   Static Proof
                 </h2>
               ) : (
-                <h2 className="text-2xl font-bold mb-8 text-center">
+                <h2 className="text-2xl font-bold mb-8 text-center mt-8">
                   Interactive Proof
                 </h2>
               )}
               <div className="grid grid-cols-2">
-                <div className="col-start-1 justify-start">
+                <div className="col-start-1 mr-[50px] w-[700px]">
+                  <span>
+                    You've completed all the geometry questions! Please answer
+                    the following questions based on your experiences with the
+                    (interactive/static) proofs in this picture.
+                    <br />
+                    <br />
+                    The word "system" in the questions mean the proof and not
```

Comment:
Maybe we should be slightly more specific, something like "The "system" includes the proof and how its information (the statements, reasons, diagrams, etc.) was visually presented to you. _It does not include the test questions._" or "Anything visible in the screenshot is part of the "system" in this context. It **does not** include the test questions."